### DISCUSSION TITLE
What are the typical IP address ranges for Data Management Zone?

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-management-zone#248] [Q&A] [answered]
What are the typical IP address ranges for Data Management Zone? In the template default is /16 (4096 IP addresses) which looks too much. 

Why do we consider such large address range and what are the main services which consumes the IPs?

**Comment by marvinbuss** [comment, marked as answer]
With the default setup, we are just consuming 11 IP addresses. Depending on the third-party tools used, additional address space may be required.  In summary that means that the following is required:
1. w/o Firewall: /27 + additional IPs for potential automation and 3rd party tools
2. w/ Firewall: at least /25 + additional IPs for potential automation and 3rd party tools
